my suggestions to brdrline trigger assignments

beta-tester opened the pull request:
according to #7328 and #7091 i would change the triggers for the inputs a little bit to fit the sounds more to what i remember to.
its only suggestions.

aaronsgiles commented on src/mame/audio/vicdual.cpp:
```diff
@@ -558,14 +558,34 @@ void borderline_audio_device::device_add_mconfig(machine_config &config)
+       or is it only in my head?
+#endif
+	NETLIST_LOGIC_INPUT(config, m_input_line[5], "GUN_TRG.IN", 0);
 	NETLIST_LOGIC_INPUT(config, m_input_line[6], "JEEP_ON.IN", 0);
```
Can you please create aliases in the netlist called I_SOUND_0, I_SOUND_1, etc, which will directly map to the m_input_line[0], m_input_line[1], etc? This way, we can leave the C code alone and just change the mapping in the netlist itself (and I would move the above comment to the netlist). When we add netlists for more vicdual games, we can use a generic sound device that doesn't understand the mapping and just connects the input bits to generic-named inputs.

beta-tester replied:
done in commit 566ff67b4665382aeab4a75134dc86a7ff684b7c below

DavidHaywood commented:
somebody is reporting there should be a startup melody, and that it was present in an old sample set.  are they misremembering, or is something like that still missing?

beta-tester commented:
@aaronsgiles ,
1. i moved the trigger input mapping from vicdual to nl_brdrline file.
2. i used the mapping from the schematics as you showed me. i didn't realized this part of the schematics (some parts are hard to read because of the low resolution).
but i am still confused - in my head i have an other mapping, so i added my suggested mapping as alternative (#if #else).
3. i included your suggestion to lower the freq for the noise gen. (i hope it is included correctly there)

@DavidHaywood , i can't remember if there was a melody when inserting a coin. in the schematics i can not see any hint of that. maybe it was an extra sound board directly connected to the coin mechanisim.

from my memory i have also a slightly different sound in my head for crawling through the field in section 2 & 3. the ANIMAL_SOUND is close to that but not 100% right what i have in mind.

i double checked the netlist but can't find an error in it.
maybe i was playing a modified version of the arcade Borderline in the early 1980'th or my head remembers to things that are not real...

beta-tester commented on src/mame/audio/nl_brdrline.cpp:
```diff
@@ -10,9 +10,17 @@
+//      in ANIMAL_SOUND
+//
+//  * noise generation with MM5837_DIP eats a lot of CPU power 
+//
```
done in commit 566ff67b4665382aeab4a75134dc86a7ff684b7c below

beta-tester commented on src/mame/audio/vicdual.cpp:
```diff
@@ -558,14 +558,34 @@ void borderline_audio_device::device_add_mconfig(machine_config &config)
+	NETLIST_LOGIC_INPUT(config, m_input_line[7], "EMAR_TRG.IN", 0);
+	NETLIST_LOGIC_INPUT(config, m_input_line[2], "WALK_TRG.IN", 0);
+	NETLIST_LOGIC_INPUT(config, m_input_line[3], "CRY_TRG.IN", 0);
 
```
done in commit 566ff67b4665382aeab4a75134dc86a7ff684b7c below

aaronsgiles commented on src/mame/audio/vicdual.cpp:
```diff
@@ -554,18 +554,42 @@ borderline_audio_device::borderline_audio_device(const machine_config &mconfig,
 	NETLIST_SOUND(config, "sound_nl", 48000)
+#else
+	NETLIST_SOUND(config, "sound_nl", 22050)
+#endif
```
Did you find that this made a performance difference? In general, the output rate of the audio doesn't impact the netlist execution performance, so I would leave it at 48000 unless you are showing some notable improvement.

beta-tester replied:
sorry, it was temporary. i thought it gives me a bit more performance with it, but it doesn't... did i forgot to remove it?

vidpro1 commented:
There is a start melody when you power on the pcb. The same melody plays when you start a game.
I can help out on this after I get a working build of the netlist version. It looks someone was trying to sort out the Tranquilizer Gun sample names from a long time ago (There was a conversion kit from Tranquilizer gun to Borderline) Not sure what that sound Borderline sound board looks like.

Sample names if you haven't used them already for netlist conversions.

boot_and_start
coin
crashes
end_level
enginenoise
field
fire

DavidHaywood commented:
so what you're saying if we haven't emulated the borderline sound board at all? only the one used for the tranqulilzer gun conversion (where we don't even have a complete dump of the romset as one ROM was bad on the PCB it was dumped from)

Osso13 commented:
AFAIK he's referring to samples Corrado recorded from a Borderline bootleg board, which might be a direct clone of the original or not, who knows.
